src/Nethermind/Nethermind.Runner/configs/chiado.cfg

```diff
@@ -30,6 +30,22 @@
     "TxPriorityContractAddress": "0x4100000000000000000000000000000000000000",
     "ForceSealing": true
   },
+  "Shutter": {
+    "Validator": "true",
+    "ValidatorRegistryContractAddress": "0x06BfddbEbe11f7eE8a39Fc7DC24498dE85C8afca",
+    "SequencerContractAddress": "0xAC3209DCBced710Dc2612bD714b9EC947a6d1e8f",
+    "KeyBroadcastContractAddress": "0xDd9Ea21f682a6484ac40D36c97Fa056Fbce9004f",
+    "KeyperSetManagerContractAddress": "0x6759Ab83de6f7d5bc4cf02d41BbB3Bd1500712E1",
+    "KeyperP2PAddresses": [
+      "/ip4/164.92.188.205/tcp/23000/p2p/12D3KooWPjX9v7FWmPvAUSTMpG7j2jXWxNnUxyDZrXPqmK29QNEd",
+      "/ip4/164.92.188.205/tcp/23001/p2p/12D3KooWJGopzaxs5G2FCPPMAy9zUoEdp5X7VebANvHGwiKW61Ck",
+      "/ip4/164.92.188.205/tcp/23002/p2p/12D3KooWPV5c2K5oHugFFArAwaJfxUrrTEjjwJgCQ44LNTsi7p68",
+      "/ip4/164.92.188.205/tcp/23003/p2p/12D3KooWKZBm2JvZE9SyPy3bQupgxtHzJC4y3E7xaHX5aRVewspC"
+    ],
+    "P2PPort": "23102",
+    "InstanceID": "61",
```

Comment:
should that go to public config?

Reply:
I don't expect the user to change this so probably not?